<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ứng dụng tính hóa đơn</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Áp dụng font chữ */
QWidget {
    font-family: &quot;Comic Sans MS&quot;, &quot;Arial&quot;, sans-serif;
    font-size: 14px;
    background-color: #FFFAF3; /* Màu nền kem nhẹ */
}

/* Nút bấm */
QPushButton {
    background-color: #FFC0CB; 
    border: 2px solid #FF69B4;
    border-radius: 10px;
    padding: 5px;
    font-size: 14px;
    font-weight: bold;
    color: white;
}

QPushButton:hover {
    background-color: #FF69B4; /* Đậm hơn khi hover */
}

QPushButton:pressed {
    background-color: #FF1493;
}

/* Bảng hiển thị món */
QTableWidget {
    background-color: #FFF5EE;
    border: 1px solid #FFB6C1;
    gridline-color: #FF69B4;
}

QHeaderView::section {
    background-color: #FFB6C1;
    font-weight: bold;
    border: 1px solid #FF69B4;
}

/* Nhãn tổng tiền */
QLabel#lbTotal {
    background-color: #FFDAB9;
    border: 2px solid #FFA07A;
    border-radius: 10px;
    padding: 5px;
    color: #D2691E;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>  Tìm  </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>Tên món</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Giá tiền </string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnThemMon">
      <property name="text">
       <string>Thêm món mới</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnTinhTien">
        <property name="text">
         <string>Tính tiền</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnXoa">
        <property name="text">
         <string>Xóa món</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearAll">
        <property name="text">
         <string>Xóa hết</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="lbTotal">
      <property name="font">
       <font>
        <family>Comic Sans MS,Arial,sans-serif</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tổng tiền: 0 VND</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
